apps/timetable/admin/ui/css/admin.css

```diff
@@ -394,6 +394,12 @@
     display: block;
 }
 
+#gh-batch-edit-container > i.fa-link{
+    font-size: 18px;
+    font-weight: 400;
+    margin-right: 5px;
+}
```

Comment:
This doesn't work for multi-line descriptions. What about the following:

```css
#gh-batch-edit-container > i.fa-link {
    font-size: 18px;
    font-weight: 400;
    margin: 11px 10px 0 0;
}
```

and in the HTML add the `pull-left` class to the icon.